Content/NPSSurvey/survey.json

```diff
@@ -0,0 +1,9 @@
+{
+  "survey_title": "How's it Going?",
+  "survey_subtitle": "How likely are you to recommend Radix and the Radix Wallet to your friends or colleagues?",
+  "survey_lowestScore_label": "0 - Not likely",
+  "survey_highestScore_label": "10 - Very likely",
+  "survey_reason_heading": "What’s the main reason for your score?",
+  "survey_reason_fieldHint": "Let us know...",
+  "survey_submitButton": "Submit Feedback - Thanks!"
```

Comment:
assymetry with `_label` but no `_button` on submitButton? but I cant remember what our standard is.

Reply:
@CyonAlexRDX Changed that to camel case